Services/Restaurant/src/TT.Restaurant/appsettings.json

```diff
@@ -5,5 +5,8 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
+  "ConnectionStrings": {
+    "DbConnection": "server=localhost;database=restaurantdb;uid=root;password=yourpassword"
```

Comment:
I'd rather have the real connection string in the `appsettings.development.json` file. Here we can have a template for the string e.g. `server=server-name;database=db-name;uid=root;password=password`

Reply:
we can have the dev appsettings be in git ignore or smth

Reply:
@Slayrak please add some placeholder insread of your actual connection string here

Reply:
Done I guess